layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,226 @@
+      <li>Mario Woods</li>
+      <li>Maurice Granton</li>
+      <li>Michael Brown</li>
+      <li>Michael Dean</li>
+      <li>Mike Ramos</li>
+      <li>Miles Hall</li>
+      <li>Natosha McDade</li>
+      <li>Patrick Harmon</li>
+      <li>Philando Castile</li>
+      <li>Quintonio LeGrier</li>
+      <li>Ramarley Graham</li>
+      <li>Robert Lawrence White</li>
+      <li>Rumain Brisbon</li>
+      <li>Ryan Twyman</li>
+      <li>Samuel David Mallard</li>
+      <li>Samuel DuBose</li>
+      <li>Sandra Bland</li>
+      <li>Sean Reed</li>
+      <li>Stephon Clark</li>
+      <li>Steven Demarco Taylor</li>
+      <li>Tamir Rice</li>
+      <li>Terence Crutcher</li>
+      <li>Terrance Franklin</li>
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li>... <a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">names</a> not listed here whose lives were taken by violence and brutality.</li>
+    </ul>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+
+  <p>In Node.js, we recognize that there is still an incredible amount of work to be accomplished to address the injustices and inequalities that exist, not only in society as a whole but within the technology industry, and even within our own project. To this point, we have not done enough and recognize that we need to do more. We commit to supporting the Black members of the community by using our collective platforms to amplify your words, or by helping you in your careers, contributing to open source, and seeking other opportunities to help.</p>
```

Comment:
```suggestion
<p>There is an incredible amount of work to do to address injustices and inequalities. In society, the technology industry, and our own project, we have not done enough. We need to do more. We commit to supporting the Black members of the community. We will use our collective platforms to amplify your words.  We will help you in your careers. We will help you in contributing to open source. And we seek other opportunities to help.</p>
```

Reply:
I am -1 to weakening our assertions here. Specifically, the changes proposed remove our responsibility in this and lessens our commitment to helping.

Reply:
I'm not sure how or why (can't find the commit) but this line is now significantly different than the one @tatianamac commented on. Here's my suggestion for that line extracted from the comment which is now unfortunately marked as outdated:

```suggestion
    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change that, but we can change how we proceed moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and by financially supporting work from Black creators.</p>
    <p>The labor to dismantle white supremacy does not fall on Black folks; it falls on the rest of us. If there are Black folks looking for collaboration, amplification, mentorship, guidance, or any sort of support, we're here for you (see below).</p>
```

Reply:
> and by financially supporting work from Black creators

How do we plan to do this? I like and support the idea but do not want us making commitments we cannot follow through on.

> The labor to dismantle white supremacy does not fall on Black folks; 

I'd suggest changing this to

> The labor to dismantle white supremacy does not fall on the Black community; 

We use folks quite a bit in the copy, which can feel a bit repetitive